### DISCUSSION TITLE
Statistics about curves driven during track

### DISCUSSION THREAD

**Discussion by HannesHoBu** [devemux86/cruiser#342] [Q&A] [answered]
I want to propose the idea to have a statistic about the curves driven during a track.
It would be a fancy feature if you would see how many curves you have passed.
The information could come from the recorded gpx track.

**Comment by devemux86** [comment, marked as answer]
@HannesHoBu thanks for the idea!

It is already in my list to offer statistics about the driven route and recorded track. :slightly_smiling_face:
 Probably examining the recorded data, we could also analyze elevation, speed, etc.

Edit: the recorded track can now be analyzed in a diagram for elevation, speed, etc.

**Comment by DolomitesTom** [comment]
... average speed with and without pauses, driven kilometers per track, duration per track with and without pauses... :-)

**Comment by airald** [comment]
track or segment... there's an option to start new segment but I think is not used when displaying graphics.

**Comment by DolomitesTom** [comment]
Cool update with showing the speed along the route. Is it very complicated to also store the average speeds? And is it possible to go back to the track details when pressing the android back button after showing the position with the haircross?

**Reply by devemux86** [reply]
Now that diagrams are implemented in Cruiser, we can think about what more data we can display.

> is it possible to go back to the track details when pressing the android back button after showing the position with the haircross?

@DolomitesTom What steps do you mean? The back button on which screen?

**Reply by DolomitesTom** [reply]
I mean the standard Android back button, not within Cruiser. When the haircross appears and you then zoom in the map you don't get back to the track details.

**Reply by devemux86** [reply]
"Android" back button to return / reopen the screen that displayed the map?

Interesting suggestion, we can think about it.
It could also confuse users, when Android back button behaves differently depending the steps.

**Comment by devemux86** [comment]
Another idea is to display the track diagram at the top of the screen.
(like the route diagram in route planning)

Then it will be easier to examine the track diagram while looking the map at the same time.

**Reply by DolomitesTom** [reply]
Yes, I also had something like this in the mind, but thought it was too difficult to realize.
